frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -105,7 +105,7 @@ const PVCSummary: React.FC<PVCSummaryProps> = ({ persistentVolumeClaim }) => {
       </dd>
       <dt>{t('console-app~Status')}</dt>
       <dd>
-        <PVCStatus pvc={persistentVolumeClaim} />
+        <PVCStatus {...persistentVolumeClaim} />
```

Comment:
Why is this change needed? It seems better to pass the PVC object as a single prop.

Reply:
> Why is this change needed? It seems better to pass the PVC object as a single prop.

@spadgett I changed it because the DetailsPage component  `getResourceStatus` wouldn't accept PVCStatus `pvc` props object -`{pvc:obj}`. It's expecting `{obj}`. 
However since the `getResourceStatus` is expecting string, I will revert it, then find another way to get status for `getResourceStatus`.